prow/external-plugins/cherrypicker/server.go

```diff
@@ -491,3 +500,14 @@ func (s *Server) getPatch(org, repo, targetBranch string, num int) (string, erro
 func normalize(input string) string {
 	return strings.Replace(input, "/", "-", -1)
 }
+
+// releaseNoteNoteFromParentPR gets the release note from the
+// parent PR and formats it as per the PR template so that
+// it can be copied to the cherry-pick PR.
+func releaseNoteFromParentPR(body string) string {
+	potentialMatch := releaseNoteRe.FindStringSubmatch(body)
+	if potentialMatch == nil {
+		return ""
+	}
+	return fmt.Sprintf("```release-note\n%s\n```", strings.TrimSpace(potentialMatch[1]))
```

Comment:
Do we want to avoid setting the release note block on the cherry-pick PR  if the release note on the parent PR is `NONE`, thinking especially in the context of k/k (if we decide to use it there someday)?

I would say yes, because we require that all cherrypick PRs should have a valid non-`NONE` release note: https://github.com/kubernetes/kubernetes/pull/72127#issuecomment-447985619.

Reply:
I agree with you here